packages/AlgernonTests.package/ALGTestAction.class/instance/testWindowOperations.st

```diff
@@ -0,0 +1,14 @@
+testing
+testWindowOperations
```

Comment:
I suggest putting those two new test cases in the `ALGTestTypes` test class (category `testing window type` maybe?) as at least for me that is what we do here (rather than the the other tests in `ALGTestAction` that mostly test the creation of Algeractions). What's your opinion on that?